python/src/geolibre/geolibre.py

```
@@ -1197,6 +1198,29 @@ def add_gpkg(
             **style,
         )
 
+    def add_polyline(
+        self,
+        polyline: str | Sequence[str],
```

Ruff (0.16.1)

[error] Undefined name `Sequence`

(F821)